### DISCUSSION TITLE
How to remove empty folder?

### DISCUSSION THREAD

**Discussion by noinhotnn** [seaweedfs/seaweedfs#4549] [Q&A] [answered]
i upload the file through filer with ttl parameter when it expires only the file is deleted the path still exists. How to remove that path?
Path: http://192.168.1.100:8888/buckets/2023/01/01/BUSSG/21424/21A1111.txt?ttl=1M
![image](https://github.com/seaweedfs/seaweedfs/assets/13940206/2b80cec3-9d1b-4871-b8fb-dc9e3ac26121)
i can run
curl -X DELETE http://192.168.1.100:8888/buckets/2023/01/01/BUSSG?recursive=true
to remove it but how to automatically expires

**Comment by chrislusf** [comment, marked as answer]
Added a fix https://github.com/seaweedfs/seaweedfs/commit/0bffb3770f4096cd8b0eb1be397a9e1372aaeea0

With the fix, you would need to create the folder first with TTL.
